e2e/features/record-agreed-hearing-requirements.feature

```diff
@@ -60,6 +60,9 @@ Feature: Record agreed hearing requirements
     And I should see the text `Do not enter the reason the appellant made the request.`
     And I should see the text `The respondent will be able to see this information and must not be informed of the appellant's personal circumstances.`
     And I should see the text `Adjustment request`
+
+
+    And I record `false` for Remote hearing
```

Comment:
Possibly more descriptive to have the text as `Remote hearing requirement will not be reviewed`

Reply:
Any reason why there need to be extra lines?